app/controllers/home_controller.rb

```diff
@@ -0,0 +1,7 @@
+class HomeController < ApplicationController
+  def index
+    @movies = Movie.order(upvotes: :desc).take(10)
+    @books = Book.order(upvotes: :desc).take(10)
+    @albums = Album.order(upvotes: :desc).take(10)
```

Comment:
You could also consider using a constant here to hold the number of media you want to display. That way if it changes later on, you only need to update it in one place instead of three.